examples/codesandbox/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "react",
+    "version": "1.0.0",
+    "description": "React example starter project",
+    "keywords": [
+      "react",
+      "starter"
+    ],
+    "main": "src/index.js",
+    "dependencies": {
+      "react": "16.13.1",
+      "react-dom": "16.13.1",
+      "react-scripts": "3.4.3",
+      "text-provider": "0.0.37"
+    },
+    "devDependencies": {
+      "typescript": "3.8.3"
+    },
+    "scripts": {
+      "start": "react-scripts start",
+      "build": "react-scripts build",
```

Comment:
Can we add a README.md in the folder to help people know how to build it or run it locally ?

Reply:
Done.